docs/source/getting-started/index.adoc

```diff
@@ -5,8 +5,8 @@
 This section contains information about installing Minishift, quickstart steps, and sample
```

Comment:
What's about we take care of "line per sentence" as well. A fair bit of the docs are already written this way. It would be nice to align the rest as well.